reports/admin.py

```diff
@@ -0,0 +1,293 @@
+            "study_group_study_levels",
+            "study_group_unit_position",
+            "study_group_unit_divisions",
+            "occurrence_place_position",
+            "occurrence_place_divisions",
+            "occurrence_languages",
+            "keywords",
+        ]
+
+
+@admin.register(EnrolmentReport)
+class EnrolmentReportAdmin(admin.ModelAdmin):
+    change_list_template = "reports/admin/enrolmentreport_changelist.html"
+    form = EnrolmentReportAdminForm
+    list_display = [
+        "id",
+        "created_at",
+        "updated_at",
+        "get_sync_status",
+        "_enrolment_id",
+        "_study_group_id",
+        "_occurrence_id",
+        "linked_event_id",
+        "enrolment_time",
+        "enrolment_status",
+        "enrolment_start_time",
+        "occurrence_start_time",
+        "occurrence_end_time",
+        "occurrence_place_id",
+        "study_group_unit_id",
+        "distance_from_unit_to_event_place",
+        "study_group_amount_of_children",
+        "study_group_amount_of_adult",
+        "occurrence_amount_of_seats",
+        "occurrence_cancelled",
+        "get_study_level_labels",
+        "get_occurrence_languages",
+        "enrolment_externally",
+        "provider",
+        "publisher",
+        "get_keywords",
+    ]
+
+    list_filter = [
+        "updated_at",
+        "enrolment_start_time",
+        "enrolment_status",
+        "enrolment_externally",
+        IsOutOfSyncListFilter,
+        HasUnitIdStudyGroupListFilter,
+        StudyLevelFilter,
+        OccurrenceLanguageFilter,
+        "occurrence_cancelled",
+        HasEnrolmentListFilter,
+        HasOccurrenceListFilter,
+        HasStudyGroupListFilter,
+    ]
+    search_fields = [
+        "study_group_unit_id",
+        "occurrence_place_id",
+        "publisher",
+        "provider",
+        "keywords",
+    ]
+    date_hierarchy = "updated_at"
+    actions = (
+        "sync_enrolment_reports",
+        "sync_enrolment_reports_with_le",
+        "export_event_enrolments_csv",
+    )
+
+    def changelist_view(self, request, *args, **kwargs):
+        count_missing = EnrolmentReport.objects.count_missing()
+        if count_missing:
+            message = ngettext_lazy(
+                f"""There is {count_missing} enrolment without a report.
+                Please sync to create the missing enrolment reports!""",
+                f"""There are {count_missing} enrolments without a report.
+                Please sync to create the missing enrolment reports!""",
+                count_missing,
+            )
+            self.message_user(request, message, messages.WARNING)
+        return super().changelist_view(request, *args, **kwargs)
+
+    def get_sync_status(self, obj):
+        # TODO: Cache needed? - May be too heavy to load for each
+        return obj._enrolment.status == obj.enrolment_status
+
+    get_sync_status.short_description = _("in sync with the enrolment")
+    get_sync_status.boolean = True
```

Comment:
You could also use `@admin.display(description=_("in sync with the enrolment"), boolean=True)`

Reply:
Good to know in future. They could all be changed all at once in another ticket.